api/api_ethereum.go

```diff
@@ -1216,41 +1220,30 @@ func (args *EthTransactionArgs) ToMessage(globalGasCap uint64, baseFee *big.Int,
 		logger.Warn("Caller gas above allowance, capping", "requested", gas, "cap", globalGasCap)
 		gas = globalGasCap
 	}
-	var (
-		gasPrice  *big.Int
-		gasFeeCap *big.Int
-		gasTipCap *big.Int
-	)
-	if baseFee == nil {
+
+	gasPrice := new(big.Int)
+	if baseFee.Cmp(new(big.Int).SetUint64(params.ZeroBaseFee)) == 0 {
 		// If there's no basefee, then it must be a non-1559 execution
-		gasPrice = new(big.Int)
-		if args.GasPrice != nil {
+		if &args.GasPrice != nil {
```

Comment:
We don't need pointer notation `&` here. `GasPrice` field is already declared as pointer in `EthTransactionArgs` struct. https://github.com/kjeom/klaytn/blob/apiDynamicGasPrice/api/api_ethereum.go#L1034-L1053

Reply:
Right, thanks!

Reply:
i think this is same with the MaxFeePerGas right? 
 https://github.com/klaytn/klaytn/blob/b128a3b6cd391726c3fd3ad8be67b57a33f959de/api/api_public_blockchain.go#L697

Reply:
@JayChoi1736 Right, Thanks for review!